examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -2312,18 +2312,18 @@ server cluster OperationalCredentials = 62 {
   }
 
   struct NOCStruct {
-    fabric_idx fabricIndex = 0;
     OCTET_STRING noc = 1;
     nullable OCTET_STRING icac = 2;
+    fabric_idx fabricIndex = 254;
   }
 
   struct FabricDescriptor {
-    fabric_idx fabricIndex = 0;
     OCTET_STRING<65> rootPublicKey = 1;
     INT16U vendorId = 2;
     FABRIC_ID fabricId = 3;
     NODE_ID nodeId = 4;
     CHAR_STRING<32> label = 5;
+    fabric_idx fabricIndex = 254;
   }
 
   readonly attribute NOCStruct NOCs[] = 0;
```

Comment:
Some more uses that need fixing:
* In `src/app/zap-templates/zcl/data-model/chip/test-cluster.xml` inside the `TestFabricScoped` struct and the `TestFabricScopedEvent` event.
* In `src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml` inside the `AccessControlEntryChanged` and `AccessControlExtensionChanged` events.

Reply:
Thanks for notice